configs/endpoints_v2/gcp.yaml

```diff
@@ -3,180 +3,5 @@ head_node_type:
-  use_spot: true
-  fallback_to_ondemand: true
-- name: gpu-worker-l4-4
-  instance_type: g2-standard-48-nvidia-l4-4
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:L4": 1
-  min_workers: 0
-  max_workers: 100
-  use_spot: true
-  fallback_to_ondemand: true
-- name: gpu-worker-l4-8
-  instance_type: g2-standard-96-nvidia-l4-8
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:L4": 1
-  min_workers: 0
-  max_workers: 100
-  use_spot: true
-  fallback_to_ondemand: true
-- name: gpu-worker-v100-1
-  instance_type: n1-standard-32-nvidia-v100-16gb-4
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:V100": 1
-  min_workers: 0
-  max_workers: 100
-  use_spot: true
-  fallback_to_ondemand: true
-- name: gpu-worker-a100-40g-1
-  instance_type: a2-highgpu-1g-nvidia-a100-40gb-1
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:A100-40G": 1
-  min_workers: 0
-  max_workers: 100
-- name: gpu-worker-a100-40g-2
-  instance_type: a2-highgpu-2g-nvidia-a100-40gb-2
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:A100-40G": 1
-  min_workers: 0
-  max_workers: 100
-- name: gpu-worker-a100-40g-4
-  instance_type: a2-highgpu-4g-nvidia-a100-40gb-4
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:A100-40G": 1
-  min_workers: 0
-  max_workers: 100
-- name: gpu-worker-a100-40g-8
-  instance_type: a2-highgpu-8g-nvidia-a100-40gb-8
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:A100-40G": 1
-  min_workers: 0
-  max_workers: 100
-- name: gpu-worker-a100-80g-1
-  instance_type: a2-ultragpu-1g-nvidia-a100-80gb-1
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:A100-80G": 1
-  min_workers: 0
-  max_workers: 100
-- name: gpu-worker-a100-80g-2
-  instance_type: a2-ultragpu-2g-nvidia-a100-80gb-2
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:A100-80G": 1
-  min_workers: 0
-  max_workers: 100
-- name: gpu-worker-a100-80g-4
-  instance_type: a2-ultragpu-4g-nvidia-a100-80gb-4
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:A100-80G": 1
-  min_workers: 0
-  max_workers: 100
-- name: gpu-worker-a100-80g-8
-  instance_type: a2-ultragpu-8g-nvidia-a100-80gb-8
-  resources:
-    cpu:
-    gpu:
-    memory:
-    object_store_memory:
-    custom_resources:
-      "accelerator_type:A100-80G": 1
-  min_workers: 0
-  max_workers: 100
-gcp_advanced_configurations_json:
```

Comment:
@akshay-anyscale is it ok to remove this?

Reply:
I'm assuming these are defaulted in @shomilj 's serverless config? If not, we should keep these

Reply:
Shomil can you confirm?

Reply:
I took one more look at the code and realized the A100-80G / A100-40G resources are known to the Autoscaler but we are not making them known to Ray (so my simple tests of asking @ray.remote(accelerator_type="A100-80G") tried to launch them, as expected, but if they actually had launched successfully, then the tasks wouldn't have been scheduled there, since we didn't set RAY_OVERRIDE_RESOURCES).

So I have one more fix to make before we can merge this PR. Doing it now.